devops/pypi-release-new.yml

```diff
@@ -0,0 +1,102 @@
+# Simplified PyPI release pipeline
+
+# At queue time, the user selects a Test or Production deployment. The following stages
+# then Run
```

Comment:
The sentence "The following stages then Run" seems odd to me. Did you mean to write "run" with lowercase R or something entirely different?

Reply:
Yes, I meant lower case 'r' - I've fixed it.